pkgs/tools/package-management/poetry/default.nix

```
@@ -30,6 +30,7 @@ let
     poetry-audit-plugin = callPackage ./plugins/poetry-audit-plugin.nix { };
     poetry-plugin-export = callPackage ./plugins/poetry-plugin-export.nix { };
     poetry-plugin-up = callPackage ./plugins/poetry-plugin-up.nix { };
+    poetry-plugin-poeblix = callPackage ./plugins/poetry-plugin-poeblix.nix { };
```

```suggestion
    poeblix = callPackage ./plugins/poeblix.nix { };
```
and even before that it wasn't sorted alphabetically.